mule-extensions-api-persistence/src/test/resources/metadata/success-result-operation-descriptor.json

```diff
@@ -894,6 +897,7 @@
             "isComponentId": false,
             "valueProviderModel": {
               "actingParameters": [],
+              "parameters": [],
```

Comment:
this is invalid and should dissapear per comments in the mule-api PR

Reply:
I'm not following you, could you explain it?

Reply:
I think I put the comment on the wrong line.. Bottom line is make sure that a strict correlation must exist between the contents of actingParameters and parameters properties. There were cases in which that wasn't happening.